apps/examples/express/src/app.ts

```diff
@@ -0,0 +1,72 @@
+import express, { Request, Response } from "express"
+import logger from "morgan"
+import * as path from "path"
+
+import {
+  errorHandler,
+  errorNotFoundHandler,
+} from "./middleware/error.middleware.js"
+
+import {
+  authenticatedUser,
+  currentSession,
+} from "./middleware/auth.middleware.js"
+import { ExpressAuth } from "@auth/express"
+import { authConfig } from "./config/auth.config.js"
+import * as pug from "pug"
+
+// Create Express server
+export const app = express()
+
+// Express configuration
+app.set("port", process.env.PORT || 3000)
+
+// Set up views engine and path
+// @ts-ignore (See https://stackoverflow.com/questions/45342307/error-cannot-find-module-pug)
+app.engine("pug", pug.__express)
+app.set("views", path.join(__dirname, "../views"))
+app.set("view engine", "pug")
+
+// Trust Proxy for Proxies (Heroku, Render.com, etc)
+// https://stackoverflow.com/questions/40459511/in-express-js-req-protocol-is-not-picking-up-https-for-my-secure-link-it-alwa
+app.enable("trust proxy")
```

Comment:
Auth.js already has a `trustHost: boolean` option in its config. Isn't that sufficient?